pkg/storaged/block/format-dialog.jsx

```diff
@@ -154,6 +156,17 @@ export function format_dialog(client, path, start, size, enable_dos_extended) {
     }
 }
 
+function find_root_fsys_block() {
+    const root = client.anaconda?.mount_point_prefix || "/";
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19854-20240118-161616-eaf4cdd4-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)